no-computer.md

```diff
@@ -11,41 +11,74 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+I believe Lego can be a DSL. The need for Lego is met by people's desires to
+build something of their own design with lego blocks. This can be thought of 
+as running a program (the end product) with pre-defined functions (the lego 
+blcoks).
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Lego is very domain-specific, in a way that it let the users use their 
+creativity to build something with ease since the lego blocks are 
+pre-manufactured. 
+So the user does not have to focus on the details of small pieces of their 
+big picture design.
```

Comment:
It's amazing how many possibilities there are with Legos given enough pieces, time, and creativity. 

Furthermore, I think Legos would be considered a DSL versus a GPL since the end result, although varied in forms, are fundamentally models and are not necessarily functioning pieces.